manifests/qualys_eus_reporting.pp

```diff
@@ -0,0 +1,50 @@
+# @summary Allow qualys to sudo to root and add wrapper for subscription-manager to lie about enabled repos.
```

Comment:
Again this no longer allows sudo, but needs sudo/escalated scan set.

Reply:
Documentation updated